spark/src/main/scala/geotrellis/spark/triangulation/BoundaryDelaunay.scala

```diff
@@ -0,0 +1,426 @@
+package geotrellis.spark.triangulation
+
+import geotrellis.vector.{Extent, Line, MultiPolygon, Point, Polygon, RobustPredicates}
+import geotrellis.vector.triangulation._
+
+import com.vividsolutions.jts.algorithm.distance.{DistanceToPoint, PointPairDistance}
+import com.vividsolutions.jts.geom.Coordinate
+
+object BoundaryDelaunay {
```

Comment:
It is of no use outside `geotrellis.spark`.  It exists to support `StitchedDelaunay`, which very much belongs in `spark`